### DISCUSSION TITLE
Contributions not showing in Profile page

### DISCUSSION THREAD

**Discussion by Ishwor-git** [community/community#60160] [Repositories]
### Select Topic Area

Bug

### Body

I have been committing daily in GitHub since last month. But recently since July 1 GitHub is not showing my contributions in my profile page.Neither private nor public contributions How can I solve this problem ?

**Comment by GhostOf0days** [comment]
Hello. Did you change your Github email address by chance? If so, please add it back to your Github account. It doesn't have to be the primary email address.

**Comment by PrachiChauhan23** [comment]
I am also facing the same issue. Can u plz help me out.

**Comment by Thorwine** [comment]
Same problem here! Contributed on Jul 5 twice, but Overview is not coloring the dot of that day 😞

**Comment by dr-stringfellow** [comment]
Same problem here ... I did not change my email address. This feature is quite important for many people so a fix would be great.

**Comment by dr-stringfellow** [comment]
Hi, I think 5-7 days are already quite a while. I know that in principle things work - I see in other profiles that private contributions show up. It only appears to affect some users.

**Comment by Thorwine** [comment]
I think I solved the issue on my side. I have no clue what happened to my commit email adress and why, but after I checked it with `git config --global user.email` the **@** sign was missing! Instead of **myName@myHost.com** it was only **myNamemyHost.com** 🤔 So I used the command `git config --global user.email "YOUR_EMAIL"` again and everthing works as usual now.

Hope this helps some of you too...
